Azure/azure-service-operator
# Bug: Found an issue with Azure SQL Server where it doesnt recognize existing SQL servers

closes #783 

The Azure SQL Server operator should detect if a SQL server exists in the selected resource group and then complete reconciliation. The code was blocking that functionality. This bug fix addresses that issue.

To test:

1. create new SQL servers from scratch (that works with out issue)
2. create a SQL server in the portal and then configure your YAML to point to that SQL server - it should reconcile almost immediately

![giphy](https://user-images.githubusercontent.com/32373900/76897396-4d828980-6859-11ea-9ae8-f603b5a8c1f9.gif)
